third_party/move/move-vm/runtime/src/tracing.rs

```diff
@@ -69,16 +67,7 @@ pub(crate) fn trace(
 ) {
     if *TRACING_ENABLED {
         let f = &mut *LOGGING_FILE.lock().unwrap();
-        writeln!(
```

Comment:
`processId-threadId` is used as the key in the coverage map. However, it is not used when generating the coverage info and printing this time-consuming. So it is removed from the trace file. bytecode instruction is also removed due to the same reason